src/tools/fuzzing/heap-types.cpp

```diff
@@ -759,7 +759,8 @@ void Inhabitator::markExternRefsNullable() {
     auto children = type.getTypeChildren();
     for (size_t i = 0; i < children.size(); ++i) {
       auto child = children[i];
-      if (child.isRef() && child.getHeapType() == HeapType::ext &&
+      if (child.isRef() && child.getHeapType().isBasic() &&
+          child.getHeapType().getBasic(Unshared) == HeapType::ext &&
```

Comment:
Perhaps we should have a helper for this common pattern of `x.isBasic() && x.getHeapType.getBasic(Unshared) == Y`, something like `x.isEqualToUnsharedBasic(Y)`?

Reply:
sgtm